browser-ext/background.js

```
@@ -0,0 +1,138 @@
+
+    extensionAPI.runtime.onInstalled.addListener(() => {
+      console.log("Smells Like Job Spirit installed!");
+      this.initializeStorage();
+    });
+  }
+
+  async handleMessage(request, sender, sendResponse) {
+    try {
+      switch (request.action) {
+        case "parseCV":
+          const result = await this.parseCV(request.file);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)